testem.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  test_page: 'tests/index.html?hidepassed',
+  test_page: 'tests/index.html?hidepassed&nolint',
```

Comment:
I preemptively added the `nolint` flag so that `yarn test` focuses on testing files (and `yarn lint` on linting files).

The `nolint` flag isn't needed for `eslint`-related packages. It is needed, however, should we later install `ember-cli-dependency-lint`.